.travis.yml

```diff
@@ -1,11 +1,20 @@
 language: python
 sudo: false
-python:
-  - "2.7"
-  - "3.4"
-env:
-  - IPYTHON=3
-  - IPYTHON=4
+
+matrix:
+  include:
+    - python: 2.7
+      env: IPYTHON=3
+    - python: 2.7
+      env: IPYTHON=4
+
+    - python: 3.4
+      env: IPYTHON=3
+    - python: 3.4
+      env: IPYTHON=4
+
+    - python: 3.6
```

Comment:
Does IPython 6 require 3.6 or higher? My inclination would be to test on the oldest supported version of Python, since that's the most likely to surface version compatibility bugs.

Reply:
It does not, though it's probably a good idea to test a newer version of python too.